llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/Passes/TransformInterpreter.cpp

```diff
@@ -38,7 +38,7 @@
 using namespace mlir;
 
 static llvm::cl::opt<std::string> clTransformFileName(
-    "linalg-transform-file-name",
+    "codegen-transform-dialect-file-name",
```

Comment:
Fine for now, but we're going to want to fix/generalize the way this is configured before using by default.

Reply:
same here re batch reply and larger design Qs